.github/workflows/ci.yml

```
@@ -29,13 +29,13 @@ jobs:
           set -xe
           python -VV
           python -m site
-          python -m pip install --upgrade pip setuptools wheel coverage[toml] virtualenv tox tox-gh-actions
+          python -m pip install --upgrade pip wheel coverage[toml] virtualenv tox tox-gh-actions
```

I'm not sure what CI is doing but I think this would also be unnecessary now:

```suggestion
          python -m pip install --upgrade pip coverage[toml] virtualenv tox tox-gh-actions
```